packaging/windows/clisdk/bundle.wxl

```diff
@@ -51,4 +51,11 @@ Ready? Set? Let's go!</String>
   <String Id="FilesInUseDontCloseRadioButton">&amp;Do not close applications. A reboot will be required.</String>
   <String Id="FilesInUseOkButton">&amp;OK</String>
   <String Id="FilesInUseCancelButton">&amp;Cancel</String>
+  <String Id="FirstTimeWelcomeMessage">Welcome to .NET Core!
```

Comment:
We need to improve how this text is displayed by adding headers, line breaks and making URLs links. But we can do that in a separate PR.